editor/editor_log.cpp

```diff
@@ -151,21 +147,15 @@ EditorLog::EditorLog() {
 	clearbutton->set_text(TTR("Clear"));
 	clearbutton->connect("pressed", this, "_clear_request");
 
-	ec = memnew(Control);
-	vb->add_child(ec);
-	ec->set_custom_minimum_size(Size2(0, 180) * EDSCALE);
-	ec->set_v_size_flags(SIZE_EXPAND_FILL);
-
-	pc = memnew(PanelContainer);
```

Comment:
same here the panel has the advantage of allowing transparent backgrounds. (for the RTL)

Reply:
and there is no problem here since, now RichText labels do not use the script editor background.
only the code editor and editor help uses it

Reply:
So the log bg cant get changed anymore? Thats fair i guess